### PR TITLE
Bugs: fetch social data from cms

### LINKED ISSUE #198: FeedAHelp: Get Social connection from CMS
Image

### PR DESCRIPTION
**Pull request by biplob2358**
# Description
fetch social data from cms

Fixes #198 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
Closes #198 
image

### CODE REVIEW

File: apps/feedahelp/src/components/ui/features/footer/FooterSocial.tsx [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
-          src={"/static/images/feedahelp/twitter.svg"}
-          width={40}
-          height={40}
-          alt={"feedahelpLogo"}
-          className="h-full object-cover"
-        />
-      </a>
+      {socialMedia.map((item: any, key) => {
+        return (
+          <a
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/footer/FooterSocial.tsx]: use next/link: 

```
import Link from 'next/link'
 
export default function Page() {
  return <Link href="/dashboard">Dashboard</Link>
}
```

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
one comment

File: apps/feedahelp/src/components/ui/features/footer/FooterSocial.tsx (hunk 1 above)
Comment (biplob2358) [reply apps/feedahelp/src/components/ui/features/footer/FooterSocial.tsx]: Done

**Review by TanvirAlam — APPROVED** [review]

**Review by urmirahman — APPROVED** [review]
LGTM

**Pull request merged**